swag/utils.py

```diff
@@ -1,5 +1,6 @@
 import math
 import random
+from swag.db import Cagnotte, CagnotteInfo
```

Comment:
`from .db`

Reply:
Resolved in : https://github.com/BobbySoftwares/Bybbot/pull/7/commits/7c244628fa49110a6b192d2cb505fcc743561694